CHANGELOG.md

```diff
@@ -2,21 +2,136 @@
 
 ## Unreleased
 
+Current changelog is accurate up to 8371b92075c90a689b6eb0a094bc6f15c19f364a
+
 - Components
     - General
+        - Update MSRV to 1.83 (unicode-org#7066)
+        - Bump default toolchain (not MSRV) to 1.89 (unicode-org#6818)
+        - Remove unused dependencies (unicode-org#6978)
+        - Use `#[expect]` for clippy lints (unicode-org#6700)
+        - Use `serde_core` where possible (unicode-org#6991)
     - `icu_calendar`
-      - Fix `und-SA-u-ca-islamic` (unicode-org#6736)
+        - Add Date::try_from_fields for flexibly building Temporal dates (unicode-org#6910)
+        - (unstable) Implement date arithmetic according to Temporal specification (unicode-org#6992, unicode-org#7012)
+        - Add a lot more documentation on individual calendars (unicode-org#7016, unicode-org#7033, unicode-org#7036, unicode-org#7037, unicode-org#7047, unicode-org#7082)
+        - Add Easter holiday to `Gregorian` and `Julian` (unicode-org#6899)
+        - Implement `PartialOrd` for `Date` unconditionally (unicode-org#7090)
+        - Add first weekday preference to week preferences (unicode-org#6615)
+        - Collapse `Dangi` and `Chinese` into `LunarChinese`, expose (unstable) customization trait (unicode-org#6938)
+        - Collapse Hijri calendar types, expose (unstable) `HijriSighting` trait for customization (unicode-org#6935)
+        - Make `extended_year` quite consistent (unicode-org#6800, unicode-org#6937)
+        - Fix `und-SA-u-ca-islamic` (unicode-org#6736)
+        - Add RataDie::in_well_behaved_astronomical_range(), use to avoid panics (unicode-org#6876)
+        - Improve some Gregorian calendar code (unicode-org#6870)
+        - Fix Hebrew MonthDay resolution with M05L (unicode-org#6966)
```

Comment:
Nit: We don't need to document fixes on previous larger changes. If you want you can move this link to be in the list of links of try_from_fields.